lib/ansible/modules/network/nxos/nxos_igmp_interface.py

```diff
@@ -442,12 +457,12 @@ def get_igmp_interface_defaults():
     return default
 
 
-def config_default_igmp_interface(existing, delta, found_both, found_prefix):
+def config_default_igmp_interface(existing, delta):
     commands = []
     proposed = get_igmp_interface_defaults()
     delta = dict(set(proposed.items()).difference(existing.items()))
     if delta:
-        command = config_igmp_interface(delta, found_both, found_prefix)
+        command = config_igmp_interface(delta, existing, None)
```

Comment:
Can you please do `command = config_igmp_interface(delta, existing, exisitng_oif_prefix_source=None)` so that it is maintainable in future?